contracts/mocks/IncreasingPriceCrowdsaleImpl.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity ^0.4.18;
+
+import "../crowdsale/price/IncreasingPriceCrowdsale.sol";
+import "../math/SafeMath.sol";
+
+
+contract IncreasingPriceCrowdsaleImpl is IncreasingPriceCrowdsale {
+
+  function IncreasingPriceCrowdsaleImpl (
+    uint256 _startTime,
+    uint256 _endTime,
+    address _wallet,
+    ERC20 _token,
+    uint256 _initialRate,
+    uint256 _finalRate
+  ) public
```

Comment:
can we follow the sytnax we use across the project for this, here and across this PR? It would look like:

```
  function IncreasingPriceCrowdsaleImpl (
    uint256 _startTime,
    uint256 _endTime,
    address _wallet,
    ERC20 _token,
    uint256 _initialRate,
    uint256 _finalRate
  )
    public
    Crowdsale(_initialRate, _wallet, _token)
    TimedCrowdsale(_startTime, _endTime)
    IncreasingPriceCrowdsale(_initialRate, _finalRate)
  {
  }
```

(where the visibility modifier is on an equal indentation with the rest of the modifiers)